test_kcidb.py

```diff
@@ -242,10 +244,10 @@ def test_upgrade_main(self):
                             stderr_re=".*Invalid major version number: '0'\n")
         self.assertExecutes(latest_version,
                             "kcidb.upgrade_main",
-                            str(kcidb_io.schema.LATEST.major + 1),
+                            str(io_latest_version.major + 1),
                             status=2,
                             stderr_re=".*No schema version found for major "
-                            f"number {kcidb_io.schema.LATEST.major + 1}\n")
+                            f"number {io_latest_version.major + 1}\n")
```

Comment:
There's plenty more references to the latest schema in the kcidb code, both direct (through `kcidb_io.schema.LATEST`) and indirect (e.g. through calling functions in `kcidb_io` and `kcidb_io.schema`, which all refer to the latest schema). We gotta update all these cases.

Reply:
Okay, I will check them and update

Reply:
@spbnick 
Kindly review my changes